### DISCUSSION TITLE
the output type of pybamm.parameter_sets

### DISCUSSION THREAD

**Discussion by xyyrly** [pybamm-team/PyBaMM#2879] [Q&A]
Hi folks,

I am a beginner in Pybamm and really need your help! When I run the code below , it returned a string"Xu2019" instead of a dictionary. What func should I use if I want a dictionary like the picture shows? Thanks a lot!

 import pybamm
 chemistry=pybamm.parameter_sets.Xu2019

![chemistry](https://user-images.githubusercontent.com/117139564/232373758-8145ff33-8f30-4e02-937f-fa250f3749c1.png)

**Comment by TomTranter** [comment]
Try going through our tutorials https://github.com/pybamm-team/PyBaMM/tree/develop/examples/notebooks/Getting%20Started
#4 deals with parameters. The syntax is 
```
parameter_values = pybamm.ParameterValues("Xu2019")
```

**Reply by xyyrly** [reply]
Thank you very much! ! ! I tried pybamm.ParameterValues and got a dictionary with all of parameters. But I want to get a dictionary with keys like 'Cell', 'seperator', 'positive electorde' instead of like 'Cell thermal expansion coefficient [m.K-1]' . Is it allowed to change values like the following?

    chemistry = pybamm.parameter_sets.Xu2019 
    chemistry["cell"] = "li_metal_Hein2020"
    chemistry["negative electrode"] = "li_metal_Hein2020"
    chemistry["positive electrode"] = "NMC622_Hein2020"
    chemistry["separator"] = "separator_Hein2020"
